docs/contributors/contributors-guide.md

```diff
@@ -35,41 +40,45 @@ Your profile is a great resource for showcasing your contributions and sharing y
 
 ![your profile](../../static/img/profile.png)
 
-On your profile, you will see the "Highlights" section, which you can learn more about [in this section](#highlights-connecting-and-recognizing-contributions), and three other tabs:
+On your profile, you will see the "Highlights" tab, which you can learn more about [in this section](#highlights-connecting-and-recognizing-contributions), and three other tabs:
 
-- **Contributions**: This tab includes a graph showing the number of pull requests (PRs) you have opened, [the velocity of your PRs](../welcome/glossary.md#pr-velocity), and the number of repositories that you have contributed to. Underneath the graph, there is a "Latest PRs" chart, which shows a list of current and recent PRs. This tab is helpful if you want to showcase a steady and consistent open source contribution experience.
+- **Contributions**: This tab includes a graph showing the number of PRs you have opened, [the velocity of your PRs](../welcome/glossary.md#pr-velocity), and the number of repositories you have contributed to. Underneath the graph is a "Latest PRs" chart, which shows a list of current and recent PRs. This tab is helpful if you want to showcase a steady and consistent open source contribution experience.
 - **Connections**: For pro accounts, you can reach out to other contributors and maintainers to collaborate on projects or to learn from them.
 - **Recommendations**: This tab includes a list of open source projects that are chosen based on the programming languages you picked in the "Interests" section of your public profile. It is useful if you're looking for new projects to contribute to.
 
 ## Explore: Finding Your Next Open Source Project
 
 The "Explore" tab is a great place to start if you're looking for a new open source project to contribute to or want to find repositories for potential job opportunities. Here, you can find a list of open source projects and filter them by language, top repos, recent, most active, and more.
 
-Once you've selected your filters, you'll be able to see the contributors, spam, and activity levels of the repository. Understanding the activity level and PR velocity of a repository can help you determine if it's a good fit for you to contribute to and what to expect when you submit a PR.
+Once you've selected your filters, you'll be able to see the contributors, spam, and activity levels of the repository. Understanding a repository's activity level and PR velocity can help you determine if it's a good fit for you to contribute to and what to expect when you submit a PR.
```

Comment:
Just a note:

We might want to revisit this. I'm not sure here are we talking about the higher view (Contributors, Spam, Merged PRs, and Unlabeled PRs) or the data in the Repositories.